wordpress-mobile/WordPress-Android
# [Blogging prompts v1] Add blogging prompts onboarding button to compact card

<img width="363" alt="image" src="https://user-images.githubusercontent.com/14964993/172745177-817537e6-cef0-4493-8ace-3a8261fb0067.png">

Fixes #

To test:
- Enable blogging prompts feature flag
- Tap on "My Site" in bottom navigation
- Tap on the FAB
- "?" button should be seen next to "Prompts" title
- Tap on "?" button
- Blogging prompts onboarding should be opened

## Regression Notes
1. Potential unintended areas of impact
None

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual testing and updated unit tests

3. What automated tests I added (or what prevented me from doing so)
Updated `WPMainActivityViewModelTest.kt`

PR submission checklist:
